src/Controls/src/Core/PointerEventArgs.cs

```diff
@@ -10,14 +10,17 @@ public class PointerEventArgs : EventArgs
 	{
 
 		Func<IElement?, Point?>? _getPosition;
+#pragma warning disable RS0016 // Add public types and members to the declared API
+		public PointerPlatformEventArgs? Args { get; set; }
```

Comment:
Maybe this can just be called `PlatformPointerEventArgs` and not use the extension method? Not sure we need a prop _and_ an extension...

Reply:
Yea, I think this is a good point. 

ToPlatform seemed like a good way to go before we decided to wrap the platform args with a class. 
Could just call this property `PlatformArgs` and call that good. That'll be WAY more discoverable as well.

Reply:
Yup, totally makes sense!

Reply:
Updated!